kernel/objects/scheduling-context/objects/SchedulingContext.cc

```diff
@@ -41,9 +41,15 @@ namespace mythos {
     void SchedulingContext::unbind(handle_t* ec)
     {
         ASSERT(ec != nullptr);
-        MLOG_DETAIL(mlog::sched, "unbind", DVAR(ec->get()));
+        MLOG_INFO(mlog::sched, "unbind", DVAR(ec->get()));
         readyQueue.remove(ec);
         current_handle.store(nullptr);
+        if(readyQueue.empty()){
+          MLOG_DETAIL(mlog::sched, "call freeSC");
+          pluginProcessorAllocator.pa.freeSC(&paTask, home->getThreadID());
```

Comment:
use a stub/event hook to make processor allocator optional?

Reply:
That seems to be a good idea.

Reply:
done